kong/plugins/jwt/handler.lua

```diff
@@ -39,7 +39,7 @@ local function retrieve_token(request, conf)
     end
   end
 
-  local authorization_header = request.get_headers()["authorization"]
+  local authorization_header = request.get_headers()[conf.header_name or "authorization"]
```

Comment:
please rebase on `next` instead of `master`, and then skip setting the default here, since those will be automatically set [since this pr](https://github.com/Kong/kong/pull/3756).